bin/start-xdebug.sh

```diff
@@ -0,0 +1 @@
+export XDEBUG_MODE=debug XDEBUG_SESSION=1 PHP_IDE_CONFIG="serverName=host.docker.internal"
```

Comment:
This bin folder is only used for the CLI commands, I would place this command inside the `docker` folder, which also contains a `config.sh`